examples/coprocessor-surrogate-cache-key/README.md

```diff
@@ -0,0 +1,124 @@
+## Context
```

Comment:
I'm not sure I get this example. I had thought that the coprocessor would send the surrogate key mapping to an external store so that invalidation could take place later but this isn't what this workflow describes. 

Am I misunderstanding?

Reply:
I think we're aligned on this. I might not understand your question

Reply:
We aligned